### PR TITLE
WIP source build

### PR DESCRIPTION
**Pull request by step21**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -3,3 +3,7 @@ max_py_ver: '37'
 max_r_ver: '35'
 conda_forge_output_validation: true
 provider: {linux_aarch64: default}
+cuda_compiler_version:                    # [linux64]
+  - None                                  # [linux64]
+docker_image:                             # [linux64]
+  - condaforge/linux-anvil-cos7-x86_64    # [linux64]
```
Comment (beckermr) [inline conda-forge.yml:9]: These lines are in the wrong file. They need to be in the `conda_build_config.yaml` per the docs.

**Comment by step21** [comment]
locally, I also added ` --with-gmp-includes=$PREFIX/include` on source compile and `make HADDOCK_DOCS=NO BUILD_SPHINX_HTML=NO BUILD_SPHINX_PDF=NO all -j${CPU_COUNT` which got (after strip) the build a bit further but I didn't test it for x86_64 regression yet so I didn't want to push yet.

**Comment by step21** [comment]
@conda-forge-admin please restart ci

**Pull request closed without merging**